library/psa_crypto_storage.c

```diff
@@ -29,6 +29,20 @@
 
 #include <stdlib.h>
 #include <string.h>
+/*
+ * When MBEDTLS_PSA_CRYPTO_SPM is defined, the code is being built for SPM
+ * (Secure Partition Manager) integration which separates the code into two
+ * parts: NSPE (Non-Secure Processing Environment) and SPE (Secure Processing
+ * Environment). When building for the SPE, an additional header file should be
+ * included.
+ */
+#if defined(MBEDTLS_PSA_CRYPTO_SPM)
```

Comment:
Why is this logic repeated three times? Please put it in a common internal header.

Reply:
Added a common internal header file `psa_crypto_service_integration.h` in d8a8074d6c16a1e1f017c139bb04e2162087b8be